source/software/singularity.rst

```diff
@@ -173,6 +173,42 @@ An important advantage of definition files is that they can easily
 be shared, and improve reproducibility.
 
 
+Conda environment in a definition file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+`Conda environments
+<https://docs.vscentrum.be/software/python_package_management.html#install-python-packages-using-conda>`_
+are a convinient solution when it comes to handling own Python-dependent
+software installations. Having a containerized Conda environment is often
```

Comment:
in the middle of sentences, "Conda" shall be changed to "conda". This is also compatible with the other page about conda, too (see software/python_package_management.rst)

Reply:
We could be either compatible, or follow the official name of the software which is Conda.

Reply:
I agree with you. In its official documentation, Conda is mention as "Conda". So, the other page shall be tailored later.